src/routes/api/request.routes.js

```diff
@@ -3,41 +3,70 @@ import RequestController from '../../controllers/request.controller';
 import Authentication from '../../middlewares/authentication';
 import controllers from '../../controllers';
 import {
-    validateCommentPost,
-    validateCommentUpdate,
-    validateCommentDelete,
-    validateCommentRetrieval, validateReqRetrieve } from '../../utils/comment.validations';
+  validateCommentPost,
+  validateCommentUpdate,
+  validateCommentDelete,
+  validateCommentRetrieval,
+  validateReqRetrieve
+} from '../../utils/comment.validations';
 import {
-    isTripRequestValid,
-    isTripRequestsSearchValid,
+  isTripRequestValid,
+  isTripRequestsSearchValid,
+  handleRequestStatusUpdate,
+  handleRequestReassignment
 } from '../../utils/validations';
-    
-const { createTripRequest, placesAndVisitTimes, getListOfMyRequests } = RequestController;
+import TripRequests from '../../middlewares/requestChecker';
 
 const {
-    searchTripRequests,
-    getUserTripsStats,
-} = RequestController;
-const {
-    isUserLoggedInAndVerified
+  isUserLoggedInAndVerified,
+  isUserManager
 } = Authentication;
-
-const router = express.Router();
 const {
-    addNewComment,
-    getCommentSpecificReq,
-    updateComments,
-    deleteComment
+  checkTripRequest,
+  isNewUserAManager
+} = TripRequests;
+const {
+  addNewComment,
+  getCommentSpecificReq,
+  updateComments,
+  deleteComment
 } = controllers.CommentController;
 
-router.post('/', [isUserLoggedInAndVerified, isTripRequestValid], createTripRequest);
-router.get('/most-traveled-destinations', isUserLoggedInAndVerified, placesAndVisitTimes);
+const router = express.Router();
+
+router.post('/', [isUserLoggedInAndVerified, isTripRequestValid], RequestController.createTripRequest);
+router.get('/most-traveled-destinations', isUserLoggedInAndVerified, RequestController.placesAndVisitTimes);
```

Comment:
why does it look like you added its code?

Reply:
Code climate was saying the destructure after the imports has issues so I had to modify it like this to make it work.